### ISSUE TITLE
Add helper text for defining the Beep sound in Alerts

### ISSUE THREAD

**Issue by f3ndot** [hexchat/hexchat#2041]
I found the tooltip for *Make a beep sound on:* string ambiguous and ended up searching through the Sounds list for "Instant Message Notification". It wasn't until I googled until I realized in an FAQ that "Beep" is the event that an Alert triggers.

### Screenshot of the misleading OS-specific (Windows in this case) tooltip:

![image](https://user-images.githubusercontent.com/740289/29243999-7a8c2586-7f7b-11e7-952e-16e38d95a681.png)

I propose adding some helper text somewhere around the *Alerts* section of the settings stating how HexChat will look for Beep first, then fallback to the OS-specific default.

[Currently in HexChat](https://github.com/hexchat/hexchat/blob/2376b7fb6a939e55edc1d97ca00055a47e0df9e3/src/fe-gtk/setup.c#L406-L414), these are the OS-specific tooltips:

* Windows:
  `Play the "Instant Message Notification" system sound upon the selected events`
* Linux (libcanberra):
  `Play "message-new-instant" from the freedesktop.org sound theme upon the selected events`
* Linux:
  `Play a GTK beep upon the selected events`

In the proposed screenshot below, these sentences would tack onto the end of the helper text to show the user how HexChat falls back.

**If the userbase feels this is worthwhile, I'm happy to initiate the PR for English and French**

### Existing Setting Window:

![image](https://user-images.githubusercontent.com/740289/29244014-e0da9016-7f7b-11e7-8845-d6da8aa35f0b.png)

### Proposed Setting Window:

![image](https://user-images.githubusercontent.com/740289/29244011-d8edb4fa-7f7b-11e7-8bcd-486c7c71270d.png)

**Comment by TingPing** [comment]
I wouldn't call it misleading and I would like users to set default sounds in their OS not HexChat ideally.

I also am not sure I like making the preferences dialog more noisy.

**Comment by f3ndot** [comment]
I agree that reducing clutter in the preferences dialog is a desirable goal.

However I'm used to having unique sounds for each IM application I use daily (Telegram, WhatsApp, Slack, HexChat, etc.) and dislike the idea that for my IRC client, I need to configure my OS' sound settings. HexChat already supports sound configuration anyway so meh, too each their own 😄 

Either way, perhaps rewording the tooltip to acknowledge "Beep" is the entry to set should a user not want to configure their OS?

e.g. for windows:

```
If defined, play the "Beep" sound otherwise "Instant Message Notification" system sound upon selected events
```
